### DISCUSSION TITLE
Generating a Dataset to predict mortality

### DISCUSSION THREAD

**Discussion by Rov7** [synthetichealth/synthea#1445] [Q&A] [answered]
Hello! I'm truly fascinated by this tool, but as someone without a medical background, I'm struggling to fully grasp how Synthea functions. Essentially, I'm aiming to create a synthetic dataset that mirrors the progression of a patient's clinical and medical data. My primary objective is to forecast individuals' lifespans by harnessing this comprehensive clinical information. Acknowledging my lack of medical expertise, I may overlook crucial features, but my intention is to compile a dataset that traces a person's medical journey until their passing, and investigate the feasibility of mortality prediction using Synthea.

Do you think it is possible to generate that data set?

Thank you!!!

**Comment by dehall** [comment]
At a high level, I think you can do what you're looking for with Synthea data. Patients in Synthea are simulated from birth till death, so you can both use it to generate training records as well as create test records to evaluate and then continue simulating to see how they progress.

That said, the disease progression models in synthea are extremely simplified compared to the complexities of the real world.  So building a model using only Synthea data and then trying to make predictions on real world patients isn't going to work. The way we would generally advise to go about this is make sure that your tools can support all the right concepts as seen in Synthea data, conditions, observations, death records, etc, and then once it's ready you can pursue real data to train the real model.

**Reply by Rov7** [reply]
Hello, first of all, thank you very much for your response. I intend to do research for my university career, it is only to verify or reject a hypothesis, so I will not be rigorous nor will I intend to apply the thesis in a real scenario. The only thing I don't understand is how to parse the json into a dataframe for later analysis. Do you know if there is a guide or something I can base myself on? I thank you very much again for your help. Greetings!

**Reply by dehall** [reply, marked as answer]
If you want an approach that could translate to real data, you could use something like fhiry https://github.com/dermatologist/fhiry or fhir-pyrate https://github.com/UMEssen/FHIR-PYrate or roll your own https://mitre.github.io/fhir-for-research/modules/tabular-python

But if all you want is quick and dirty analysis, it's probably easier to use the CSV output from Synthea which exports the data in a structured format similar to a relational database. If you're generating a new dataset, enable the csv export by setting the `exporter.csv.export` setting to true, either in `./src/main/resources/synthea.properties` or on the command line `./run_synthea --exporter.csv.export=true ...` 
More info on the columns is here: https://github.com/synthetichealth/synthea/wiki/CSV-File-Data-Dictionary
